meson.build

```diff
@@ -280,7 +280,6 @@ substs.set('userenvgeneratordir',                             userenvgeneratordi
 substs.set('systemshutdowndir',                               systemshutdowndir)
 substs.set('systemsleepdir',                                  systemsleepdir)
 substs.set('CERTIFICATEROOT',                                 get_option('certificate-root'))
-substs.set('SYSTEMCTL',                                       join_paths(rootbindir, 'systemctl'))
```

Comment:
not a fan of this commit. it just makes things less deterministic, since we are subject to searching rather than fixed paths. Can we keep this independent of the other parts of the PR? the searching of executables stuff is problematic enough, I am not convinced at this point that we should make use of that for our own binaries too, needlessly...

Reply:
I pushed this initially as a separate PR. I'll just reopen that and reply to this comment there.